frontend/src/components/orders/OrderSummary.vue

```diff
@@ -0,0 +1,37 @@
+<script setup lang="ts">
+import Card from "primevue/card";
+
+import type { Order } from "@/global/types/entities";
+import { ROUTES } from "@/router";
+
+const props = defineProps<{
+  orderInfo: Order;
+}>();
+</script>
+<template>
+  <Card class="my-2">
+    <template #content>
```

Comment:
Maybe here we can also think about a design that uses the full width of the screen?

Reply:
<img width="1008" alt="image" src="https://github.com/user-attachments/assets/ecb0e785-fde7-4264-91b1-964c0cb982f5">